.github/workflows/pytest.yml

```diff
@@ -0,0 +1,23 @@
+name: PyTest
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
```

Comment:
Should we do the same for all python versions? So that we get to know fast any incompatibilities